### ISSUE TITLE
(Bug report) Paragraph Handle and CKE Toolbar scrolling issues

### ISSUE THREAD

**Issue by apg-dev** [TriliumNext/Trilium#847]
![paragraph-mark-example](https://user-images.githubusercontent.com/55888411/73553521-19bde280-4418-11ea-9648-b8ca510baf7f.gif)

**Comment by zadam** [comment]
I agree this is not great however this is the CKEditor's behavior and I don't really have a way to control that :/

I like your theme BTW ;)

**Comment by apg-dev** [comment]
Gotcha. Any chance you can enable more CKEditor options such as horizontal line and tab indent?

Ty on theme comment. Just published it: https://github.com/apg-dev/trilium-theme-bear

**Comment by zadam** [comment]
tab indent is not supported yet as far as I know.

Horizontal line - I think I evaluated it in the past and might include it in the future.

**Comment by gerroon** [comment]
@apg-dev 

Is it possible to use your theme offline?

**Comment by apg-dev** [comment]
Hello @gerroon: yes. Let me know if you run into any problems. Note that I'm using Trilium in a Mac environment (both desktop and hosted).

**Comment by eliandoran** [comment]
Solved upstream, see https://github.com/TriliumNext/Trilium/issues/2308#issuecomment-2080339508 for more info.
